src/main/java/org/folio/spring/SpringContextUtil.java

```diff
@@ -40,22 +42,28 @@ public static void init(Vertx vertx, Context context) {
    */
   public static void init(Vertx vertx, Context context, Class<?> defaultConfiguration) {
     String configClassName = context.config().getString(SPRING_CONFIGURATION);
+    log.debug("init:: Attempts to create spring base context");
+
     if (configClassName != null) {
+      log.info("init:: configClassName != null");
```

Comment:
maybe something like this? 
```suggestion
      log.info("init:: Attempts to create spring context by configClassName");
```